LayoutTests/imported/w3c/web-platform-tests/css/css-grid/grid-relayout-with-nested-grid.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<meta charset="utf-8">
+<link rel="author" title="Brandon Stewart" href="mailto:brandonstewart@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid-1">
+<link rel="match" href="grid-relayout-with-nested-grid-ref.html">
+<meta name="assert" content="Ensure text does not overflow upon re-layout of grid">
+</head>
+
+<style>
+.grid { 
+  display: grid; 
+  grid-template-columns: minmax(min-content, 100px) auto; 
+}
+
+.nested_grid {
+  border: 5px solid; 
+  display: grid; 
+  justify-items: baseline; 
+  overflow-wrap: break-word;
+}
+
+</style>
+<body>
+<div id="grid" class=grid>
+  <div id="extra_div1"> 
+    <div id="extra_div2">
+      <div id="nested_grid" class=nested_grid><p id="paragraph">This content will overflow.</p></div>
+    </div>
+  </div>
+</div>
+</body>
+
+<script>
+/* Force a re-layout */
+document.getElementById("extra_div1").appendChild(document.createElement("p"));
+document.documentElement.classList.remove("reftest-wait");
```

Comment:
reftest-wait shouldn't be necessary with the setTimeout removed. It's only useful for async operations. I assume you want the setTimeout() so this layout doesn't count as part of the initial one?

Reply:
Correct, we should be doing a second layout, so I will need to to put in the `setTimeout()` again.